assets/blocks/reader-registration/edit.js

```diff
@@ -142,6 +147,14 @@ export default function ReaderRegistrationEdit( {
 										setAttributes( { displayListDescription: ! displayListDescription } )
 									}
 								/>
+								<ToggleControl
+									label={ __( 'Hide input and always subscribe', 'newspack' ) }
```

Comment:
```suggestion
									label={ __( 'Hide newsletter selection and always subscribe', 'newspack' ) }
```

Also, IMO a help text should further describe this settings. Along the lines of "If a single list is selected in the options below, the block can hide the newsletter selection and always subscribe the new reader to the list".